src/gui/AboutDialog.cpp

```diff
@@ -20,13 +20,113 @@
 #include "ui_AboutDialog.h"
 
 #include "config-keepassx.h"
-#include "git-info.h"
 #include "core/FilePath.h"
 #include "crypto/Crypto.h"
+#include "git-info.h"
 
 #include <QClipboard>
 #include <QSysInfo>
 
+static const QString aboutMaintainers = R"(
+<p><ul>
+    <li>Jonathan White (<a href="https://github.com/droidmonkey">droidmonkey</a>)</li>
+    <li>Janek Bevendorff (<a href="https://github.com/phoerious">phoerious</a>)</li>
+    <li><a href="https://github.com/TheZ3ro">TheZ3ro</a></li>
+    <li>Louis-Bertrand (<a href="https://github.com/louib">louib</a>)</li>
+    <li>Weslly Honorato (<a href="https://github.com/weslly">weslly</a>)</li>
+    <li>Toni Spets (<a href="https://github.com/hifi">hifi</a>)</li>
+    <li>Sami V&auml;nttinen (<a href="https://github.com/varjolintu">varjolintu</a>)</li>
+</ul></p>
+)";
+
+static const QString aboutContributors = R"(
+<h3>VIP Patreon Supporters:</h3>
+<ul>
+    <li>John Cook</li>
+    <li>Max Anderson</li>
+    <li>l0b0</li>
+    <li>NarwhalOfAges</li>
+    <li>Caleb Currie</li>
+    <li>Igor Zinovik</li>
+    <li>Morgan Courbet</li>
+    <li>Sergiu Coroi</li>
+</ul>
+<h3>Notable Code Contributions:</h3>
+<ul>
+    <li>droidmonkey</li>
+    <li>phoerious</li>
+    <li>TheZ3ro</li>
+    <li>louib</li>
+    <li>weslly</li>
+    <li>varjolintu (KeePassXC-Browser)</li>
+    <li>hifi (SSH Agent)</li>
+    <li>ckieschnick (KeeShare)</li>
+    <li>seatedscribe (CSV Import)</li>
+    <li>brainplot (many improvements)</li>
+    <li>kneitinger (many improvements)</li>
+    <li>frostasm (many improvements)</li>
+    <li>fonic (Entry Table View)</li>
+    <li>kylemanna (YubiKey)</li>
+    <li>keithbennett (KeePassHTTP)</li>
+    <li>Typz (KeePassHTTP)</li>
+    <li>denk-mal (KeePassHTTP)</li>
+    <li>angelsl (KDBX 4)</li>
+    <li>debfx (KeePassX)</li>
+    <li>BlueIce (KeePassX)</li>
+</ul>
+<h3>Patreon Supporters:</h3>
+<ul>
+    <li>Ashura</li>
+    <li>Alexanderjb</li>
+    <li>Andreas Kollmann</li>
+    <li>Richard Ames</li>
+    <li>Christian Rasmussen</li>
+    <li>Gregory Werbin</li>
+    <li>Nuutti Toivola</li>
+    <li>SLmanDR</li>
+    <li>Tyler Gass</li>
+    <li>Lionel Laské</li>
+    <li>Dmitrii Galinskii</li>
+    <li>Sergei Maximov</li>
+    <li>John-Ivar</li>
+    <li>Clayton Casciato</li>
+</ul>
+<h3>Translations:</h3>
+<ul>
+    <li><strong>Basque</strong>: azken_tximinoa, Hey_neken</li>
+    <li><strong>Catalan</strong>: capitantrueno, dsoms, mcus, raulua, ZJaume</li>
+    <li><strong>Chinese (China)</strong>: Biggulu, Brandon_c, hoilc, ligyxy,
+        vc5, Small_Ku</li>
+    <li><strong>Chinese (Taiwan)</strong>: BestSteve, MiauLightouch, Small_Ku,
+        yan12125, ymhuang0808</li>
+    <li><strong>Czech</strong>: DanielMilde, JosefVitu, pavelb, tpavelek</li>
+    <li><strong>Danish</strong>: nlkl</li>
+    <li><strong>Dutch</strong>: apie, bartlibert, evanoosten, fvw, KnooL, srgvg,
+        Vistaus, wanderingidea</li>
+    <li><strong>Finnish</strong>: artnay, Jarppi, MawKKe</li>
```

Comment:
I and @hifi both are also translated a big bunch of stuff.

Reply:
I suppose i should update the translator section as well.